.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,77 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: [Bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: repro-steps
+    attributes:
+      label: Repro steps
+      description: If you can reproduce the issue, please provide detailed repro steps here or provide a link to a repro project.
+      placeholder: 1. X, 2. Y, 3. Crash!
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version of the dart/Flutter SDK are you using?
```

Comment:
@blagoev isn't this where we ask about the package version? Or do you mean something else?

Reply:
oh, I misread that.